### PR TITLE
EN-9-CARDS: Adding cards for employees

### PR DESCRIPTION
**Pull request by maxituc21**
Adding the feature to show employees as cards. Note: image is hardcoded until we have the feature to upload images.

### CODE REVIEW

File: src/employees.js [hunk 1]
```diff
@@ -29,27 +31,67 @@ const referenceValues = {
...
+                    <Grid
+                        xs={2}
+                        item
+                    >
+                        <Card sx={{ maxWidth: 300 }}>
+                            {/*Profile image hardcoded until photo upload feature is in palce*/}
+                            <CardMedia
+                                component="img"
+                                height="200"
+                                image="https://bolavip.com/__export/1625160988556/sites/bolavip/img/2021/07/01/gettyimages-1323568516_crop1625160987586.jpg_1159711837.jpg"
```
Comment (agustinb4) [inline src/employees.js:65]: I updated the image to our s3, please update de url: https://entropay-assets.s3.amazonaws.com/messi.jpg

**Review by agustinb4 — APPROVED** [review]
Looks good! Just a minor change

File: src/employees.js [hunk 2]
```diff
@@ -59,25 +101,25 @@ export const EmployeeCreate = () => (
...
 export const EmployeeProfile = () => (
-    <Show title="Show employee" emptyWhileLoading>     
-        <Grid container direction="row" justifyContent="flex-start" alignItems="center">      
+    <Show title="Show employee" emptyWhileLoading>
+        <Grid container direction="row" justifyContent="flex-start" alignItems="center">
             <Grid item>
-                <Box m={2}>
-                    <Avatar alt="Employee" src="" sx={{ width: 100, height: 100 }}  />
+                <Box m={2}>{/*Profile image hardcoded until photo upload feature is in palce*/}
+                    <Avatar alt="Employee" src="https://bolavip.com/__export/1625160988556/sites/bolavip/img/2021/07/01/gettyimages-1323568516_crop1625160987586.jpg_1159711837.jpg" sx={{ width: 100, height: 100 }} />
```
Comment (agustinb4) [inline src/employees.js:108]: same here, https://entropay-assets.s3.amazonaws.com/messi.jpg

**Review by FacuOrellana — APPROVED** [review]

File: src/employees.js (hunk 1 above)
Comment (maxituc21) [reply src/employees.js]: Done.

**Pull request merged**